.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   mac-os-build-clang:
-    runs-on: macos-12
+    runs-on: macos-13
```

Comment:
Let us change this back to 12, I have a fix for the build on 13. I can address it in another PR later

Reply:
Done